charts/elastic/1.2.1/templates/autoscaling-kibana.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.kibana.enabled }}
+{{- if .Values.kibana.params.autoscaling.enabled }}
+---
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "common.names.shortname" $ }}
+  namespace: {{ include "common.names.namespace" $ }}
+spec:
+  {{- with .Values.kibana.params }}
+  scaleTargetRef:
+    apiVersion: kibana.k8s.elastic.co/v1
+    kind: Kibana
+    name: {{ include "common.names.shortname" $ }}
+  minReplicas: 1
+  maxReplicas: {{ .autoscaling.maxReplicas }}
```

Comment:
```suggestion
  maxReplicas: {{ .autoscaling.maxNodes }}
```

Reply:
Using key names for kibana from elastic documentation:
https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-stateless-autoscaling.html